libs/backendless-console-api.js

```diff
@@ -477,6 +495,8 @@ class Backendless {
 
         app.tables = app.tables.filter(table => !['orders_dump', 'tmp'].find(prefix => table.name.startsWith(prefix)))
 
+        resolveRelationIdentificationColumns(app.tables)
```

Comment:
there is a cleanTable method, which has a cleanRelation method, so all iteration is already done for you. just add identification column resolving there